### PR TITLE
SLING-10684 add integration tests to verify

### PR DESCRIPTION
**Pull request by enapps-enorman**
For verifying the SLING-10677 fix

Requires https://github.com/apache/sling-org-apache-sling-scripting-sightly-testing-content/pull/5

### CODE REVIEW

**Comment by enapps-enorman** [comment]
NOTE: the ci build fails because of new SNAPSHOT dependencies that are not available there yet.

File: pom.xml [hunk 1]
```diff
@@ -336,13 +336,13 @@ Require-Capability: osgi.extender;filter:="(&(osgi.extender=sling.scripting)(ver
...
             <artifactId>org.apache.sling.scripting.sightly.runtime</artifactId>
-            <version>1.2.4-1.4.0</version>
+            <version>1.2.5-1.4.0-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.scripting.sightly.compiler</artifactId>
-            <version>1.2.12-1.4.0</version>
+            <version>1.2.13-1.4.0-SNAPSHOT</version>
```
Comment (raducotescu) [inline pom.xml:345]: I don't think you need to bump the compiler.

**Review by raducotescu — APPROVED** [review]
Thanks for the PR! See https://github.com/apache/sling-org-apache-sling-scripting-sightly-testing-content/pull/5#issuecomment-888865578, maybe we could push that in the same changeset?!

File: pom.xml (hunk 1 above)
Comment (raducotescu) [reply pom.xml:345]: Ah, that's for https://github.com/apache/sling-org-apache-sling-scripting-sightly-compiler/pull/8, but it was not clear from this PR, nor from https://issues.apache.org/jira/browse/SLING-10677.

Comment (enapps-enorman) [reply pom.xml:345]: > I don't think you need to bump the compiler.

You are probably right.  None of the tests here are directly impacted by https://github.com/apache/sling-org-apache-sling-scripting-sightly-compiler/pull/8 

The testing for SLING-10682 was done with local unit tests.

**Review by raducotescu — APPROVED** [review]

**Pull request merged**